contents/handbook/engineering/support-hero.md

```diff
@@ -129,10 +129,20 @@ You’ll spend most of your time in the Views pane, where you’ll find all tick
 
 Tips:
 
-* If need more information from the customer to solve the issue, respond and mark as pending. 
-* If you think you solved the issue mark as solved (if they reply it will re-open and it's easier for everyone if there's less open tickets around).
+* Err on the side of Solving tickets (see below) if you expect no further input from the customer, as a lot of them don't reply to confirm that the problem has been solved.
 * Provide actionable information as _Note_ (e.g. links to internal slack threads, partial investigation, ...)
-* Low priority tickets don't send emails to the requester and can be viewed in aggregation and closed without a public reply. High and normal priority tickets send an email about a helpdesk ticket being created, so we should respond publicly there.
+
+#### Ticket Status
+
+When responding to a ticket you should also choose an appropriate status according to the following:
+
+* **New** - A newly created ticket, you shouldn't need to use this when responding
+* **Open** - The ticket is still awaiting a response/further investigation from someone in PostHog (if it's not you make sure the other person/team knows about it).
+* **On-Hold** - We have enough information from the customer but a resolution is blocked by something internal to PostHog (e.g. query performance, PoE).  You should let them know that this is the case.
+* **Pending** - The problem isn't solved and you have asked for further information from the customer.
+* **Solved** - The problem is solved, and you don't expect to do any further work on the ticket.  If it's related to a feature request then you should provide the customer a link to GitHub so that they can follow along with development.
```

Comment:
```suggestion
* **Solved** - You've provided a solution and don't expect to do any further work on the ticket.  If it's related to a feature request then you should provide the customer a link to GitHub so that they can follow along with development.
```

I think this is clearer. Tickets can always reopen and we can now track first/full resolution time for such as case. 

If we make this so that it's "I have provided a solution" instead of "the user has confirmed the solution" then we'll also be able to track one-touch tickets better and report more accurate resolution times for cases where we solve the issue but a user doesn't confirm.